src/components/AssetTimeseriesPanel/stories/examples.story.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { decorators, customStyle, metaOptions, handleAssetTimeseriesLoaded } from './helper';
+import { AssetTimeseriesPanel } from '../AssetTimeseriesPanel';
+
+<Meta title="Assets|AssetTimeseriesPanel/Example" decorators={decorators} />
+
+## Custom Spinner
+      
+<Preview withSource='open'>
```

Comment:
@mrtraser can we define this somewhere in one place?

Reply:
I've just updated `docs/configure-addon-docs` branch with set code examples expanded by default

Reply:
so here is no need to define `withSource='open'` here